spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/BoxControllerHelperService.java

```diff
@@ -621,62 +617,34 @@ public JSONObject getBoxBarcode(HttpSession session, JSONObject json) {
    * @return JSON message indicating success or error
    */
   public JSONObject printBoxBarcodes(HttpSession session, JSONObject json) {
-    try {
-      User user = authorizationManager.getCurrentUser();
+    return ControllerHelperServiceUtils.printBarcodes(securityManager, printManager, json, new BarcodePrintAssister<Box>() {
```

Comment:
Ah, I didn't realize the reuse in `ProjectControllerHelperService`. This could still be done for  `BoxControllerHelperService`, since it isn't used in the project. Either way is fine. The cost is minimal, but there's no benefit to incurring it